src/libraries/System.Private.CoreLib/src/System/BitConverter.cs

```diff
@@ -848,7 +848,7 @@ public static unsafe short HalfToInt16Bits(Half value)
         }
 
         /// <summary>
-        /// Converts the specified 16-bit signed integer to a half-precision floating point number.
+        /// Converts the specified 16-bit unsigned integer to a half-precision floating point number.
```

Comment:
The method is accepting a `short`, which is signed.  This change should be reverted.

Reply:
Yes, my mistake.